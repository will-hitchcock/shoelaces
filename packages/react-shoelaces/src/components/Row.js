import styled from 'styled-components'
import { row } from 'shoelaces'

export const Row = styled.div`${row}`

export default Row
