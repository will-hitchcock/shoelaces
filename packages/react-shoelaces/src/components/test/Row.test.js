import Row from '../Row'

describe('<Row />', () => {
  it('should render a <Row />', () => {
    expect(shallow(<Row />)).toMatchSnapshot()
  })
})
