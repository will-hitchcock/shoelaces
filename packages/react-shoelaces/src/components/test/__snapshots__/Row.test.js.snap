// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> should render a <Row /> 1`] = `
<div
  className="sc-bdVaJa cErtkT"
/>
`;
